### DISCUSSION TITLE
When will you be releasing the CustomSpan enhancement?

### DISCUSSION THREAD

**Discussion by Pete-PlaytimeSolutions** [jufab/opentelemetry-angular-interceptor#49] [General]
Just wondering what is timeframe that you will be releasing the Custom Span functionality?
Thanks

**Comment by jufab** [comment]
Hi @Pete-PlaytimeSolutions ,

I follow Opentelemetry-js release. 
So the next release include upgrade and custom span (and issue #50 ;) )

**Reply by jufab** [reply]
@Pete-PlaytimeSolutions it's now available in version 0.20.0
